src/blog/2023/011-form-js-1.6.md

```diff
@@ -0,0 +1,101 @@
+
+As an example, if wanted to use dynamic lists as a read-only array data renderer, simply disable the ability to add/remove entries on the dynamic list, make all the child fields read-only and voilà.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list-readonly.png">
+  <p class="caption">
+    Use the component however you like, it opens up many avenues for creative use cases.
+  </p>
+</div>
+
+And the best part, you still get all the features you expect out of simple elements in your dynamic list repetitions. You may now make use of the `this` variable within FEEL expressions to refer to the local 'scope' of a dynamic list's rendered element. 
+
+As small heads up however, we are planning to make some changes to how these scopes work in the future, so keep this in mind.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-dynamic-list-simple-scope.gif">
+  <p class="caption">
+    Conditionally hide or show on a row by row basis within a dynamic list. 
+  </p>
+</div>
+
+So, while simple on the surface, the dynamic list really was the final piece of the puzzle for a great deal of applications. The ability to flexibly handle data arrays combined with binding paths and keys leads to make quick configuration and clean data outputs the default experience in form-js.
+
+Now, what if you're dealing with much larger data sets that you want to render in the application, browse through, maybe use as a reference within your form?
+
+
+## The first iteration of Tables
+
+If you need to display large datasets, for reference purpose for example, you may use the new table component.   
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-tables.gif">
+  <p class="caption">
+    Efficiently manage tabular data with the new Table Component.
+  </p>
+</div>
+
+Our first iteration already brings in plenty of functionality, namely row sorting, pagination with control over each page's height, and both static and dynamic definition of headers which means you have the ability to render your tables entirely from input data definitions, giving you a great deal of programatic control over the final appearance.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/011-tables-more.png">
+  <p class="caption">
+    We're not done yet with this component. We will iterate upon tables based on feedback, and see where we can take it.
+  </p>
+</div>
+
+
+## Our focus for 1.7
+
+As you've probably noticed, form-js has grown a lot in the last year or two. We've made a lot of experimental calls, and built a lot of new things from scratch. But we haven't had dedicated time to step off the gas yet, which has caused some performance slowdowns and some of the usual problems in growing projects. 
```

Comment:
Let's take this into the other blog. Very good to openly share our future plans.